client/src/assets/blog/next_steps.ts

```diff
@@ -0,0 +1,50 @@
+export class NextSteps {
+  static title = 'You’re at risk for lead in your water. What now?';
+  static image = 'next-steps.jpg';
+  static content = `  <p>Learning that your area is at medium or high risk for lead may sound
```

Comment:
nit: is this space before the tag necessary?

Reply:
Removed!